Dockerfile.pipelines

```diff
@@ -50,3 +68,34 @@ RUN conda update -q conda
 
 RUN echo "dash dash/sh boolean false" | debconf-set-selections
 RUN DEBIAN_FRONTEND=noninteractive dpkg-reconfigure dash
+
+RUN adduser --disabled-password --gecos "" devito
+
+COPY . /home/devito
+RUN chown -R devito /home/devito
+
+USER devito
+WORKDIR /home/devito
+
+RUN if [ $installWithPip == "true" ] ; then \ 
+      python setup.py install --user ; \
+    else \
+      conda env create -q -f environment.yml ; \
+      source activate devito ; \
+      echo $PATH ; \
+      pip install -e . ; \
+      pip install pytest-xdist ; \
+      if [ $useOpenMP != "true" ] ; then \
```

Comment:
the price to pay is introducing an extra flag, but this should actually be something like "if useMPI: then ..."

Reply:
Good thought - done.